lib/beaker-pe/install/pe_utils.rb

```diff
@@ -478,7 +478,7 @@ def verify_network_resources(hosts, network_resources)
             if network_resources != nil
               network_resources.each do |resource|
                 # curl the network resource silently (-s), only connect (-I), and don't print the output
-                on host, "curl -I -s #{resource} > /dev/null", :accept_all_exit_codes => true
+                on host, "curl -I -s --max-time 10 #{resource} > /dev/null", :accept_all_exit_codes => true
                 if host.connection.logger.last_result.exit_code != 0
```

Comment:
isn't this the result from the line just above this one? Why don't we assign the result from the `on` function call & then inspect that? Seems like a pretty convoluted way to get the result we could have in a much simpler way.